.github/workflows/post-apiview.yml

```diff
@@ -0,0 +1,29 @@
+name: After APIView
+
+on:
+  check_run:
+    types: [completed]
+    
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  event-handler:
+    name: After APIView
+    runs-on: ubuntu-24.04
+    if: ${{ github.event.check_run.check_suite.app.name == 'Azure Pipelines' && 
+          ( contains(github.event.check_run.name, 'APIView') ||
+            contains(github.event.check_run.name, '(SDK Generation') ||
```

Comment:
Do we want to have the `(` in the quoted string? If not I would remove it to avoid confusion I had to double check to make sure the other one wasn't missing.

Reply:
I think this is a typo, see example here:

https://github.com/Azure/azure-rest-api-specs/blob/4b03f0beabaf5a2d42fedb8cbb37bce41cc09344/.github/workflows/sdk-breaking-change-labels.yaml#L18

Reply:
It's a typo. @raych1  wanted it this way. In some cases, the pipeline name may contain `Preview`

Reply:
I'm ok with not having the trailing `)` but I would also remove the opening `(` to avoid an confusion in the future.